### PR TITLE
Added raw export format

### PR DESCRIPTION
**Pull request by emerencia**
This supports requests such as:

`http://physiqual.dev/physiqual/oauth_session.raw?state=google&data_source=steps`

### CODE REVIEW

File: lib/physiqual/exporters/raw_exporter.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+module Physiqual
+  module Exporters
+    class RawExporter < Exporter
+      def configure(service_provider, data_source)
+        @service_provider = service_provider
+        @data_source = data_source
+        self
```
Comment (frbl) [inline lib/physiqual/exporters/raw_exporter.rb]: ? wat is dit

Comment (emerencia) [inline lib/physiqual/exporters/raw_exporter.rb]:
```ruby
render json: Exporters::RawExporter.new.configure(params[:state], params[:data_source]).export(current_user, first_measurement, number_of_days)
```

File: lib/physiqual/exporters/raw_exporter.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        return [] unless !tokens.blank? && tokens.first.complete?
+        token = tokens.first
+        session = Sessions::TokenAuthorizedSession.new(token.token, token.class.base_uri)
+        service = DataServices::DataServiceFactory.fabricate!(token.class.csrf_token, session)
+        from, to = determine_time_span(first_measurement, number_of_days)
+        service.send(@data_source.to_sym, from, to)
+      end
+
+      def export(user, first_measurement, number_of_days)
+        result = export_data(user, first_measurement, number_of_days)
```
Comment (frbl) [inline lib/physiqual/exporters/raw_exporter.rb]: export_data(user, first_measurement, number_of_days).to_json

File: spec/lib/physiqual/exporters/raw_exporter_spec.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          result = subject.export(user, first_measurement, number_of_days)
+          expect(json? result).to be_truthy
+        end
+      end
+
+      it 'should be a subclass of Exporter' do
+        expect(described_class.ancestors).to include Exporter
+      end
+
+      def json?(json)
```
Comment (frbl) [inline spec/lib/physiqual/exporters/raw_exporter_spec.rb]: hier

**Comment by frbl** [comment]
lgtm

**Pull request merged**